xm-commons-topic/src/main/java/com/icthh/xm/commons/topic/domain/ConsumerHolder.java

```diff
@@ -0,0 +1,13 @@
+package com.icthh.xm.commons.topic.domain;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.kafka.listener.AbstractMessageListenerContainer;
+
+@Getter
+@RequiredArgsConstructor
+public class ConsumerHolder {
```

Comment:
Seems that visibility of this class can be package-private

Reply:
no, TopicManager can't get access to this class in case if package-private